api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/alerts_controller.rb

```diff
@@ -34,11 +34,9 @@ def update
     return render_bad_request('update alert attributes', 'no alert hash specified') unless params.require(:alert)
     return render_not_found('alert') unless alert
 
-    attributes = params.require(:alert).permit(:title, :webhook, :sent)
-
     authorize! :manage_alert, alert
 
-    if alert.update(attributes)
+    if alert.update(kpi_update_params)
```

Comment:
same as above, update users one by one if this query did not fill the recipients array (= mnohub still at old version)